### ISSUE TITLE
Question about loss?

### ISSUE THREAD

**Issue by yemai666** [thunlp/OpenKE#386]
How much transE lost indicates that the training was effective，For example, there are 17,289 data in my training set, 8,320 entities and 26 relationships, and the loss is 0.416 after 3000 times，I wonder if this loss is justified？

**Comment by Huda-Hakami** [comment]
The link prediction result will provide an indication whether the learnt embeddings are good.
